src/gracenotegroup.ts

```diff
@@ -90,7 +108,7 @@ export class GraceNoteGroup extends Modifier {
     this.preFormatted = false;
 
     this.show_slur = show_slur;
-    this.slur = null;
+    this.slur = undefined;
```

Comment:
changing null to undefined can be a breaking change. I've seen a lot of places with:
```javascript
if (a === null) ...
```
which is not the same as:
```javascript
if (a === undefined) ...
```

Reply:
I agree I have already changed 
```
if (a === null)
```
by 
```
if (!a)
```
If we also leave `null` the declarations, parameters and return values will get very complex.

Reply:
same problem here. I would not change that:
```javascript
if (a === null)
```
is not the same as:
```javascript
if (!a)
```

Reply:
Maybe it was not intentional @0xfe? Then it is a fine change

Reply:
This case is okay (though I agree with @wassertim that we should be careful with these.) `undefined` is semantically more correct here than `null`.